update-doi-request/src/test/java/no/unit/nva/doi/requests/handlers/ApiUpdateDoiRequestHandlerTest.java

```diff
@@ -207,12 +207,18 @@ public void handleRequestSucessfullyUpdatesStatusWhenPublicationIdIsValid() thro
         var actualUpdatedPublication = readPublicationDirectlyFromDynamo(publication.getIdentifier());
         var expectedUpdatedPublication = expectedDoiRequestSummary(publication, actualUpdatedPublication);
 
+        syncUninjectableModifiedDates(actualUpdatedPublication, expectedUpdatedPublication);
         assertThat(actualUpdatedPublication, is(equalTo(expectedUpdatedPublication)));
 
         assertThat(response.getHeaders(), hasEntry(HttpHeaders.LOCATION,
             FAKE_ENV_SCHEMA_AND_HOST + publication.getIdentifier().toString()));
     }
 
+    private void syncUninjectableModifiedDates(Publication actualUpdatedPublication,
```

Comment:
This one needs some investigation, because `expectedDoiRequestSummary` already ensures that clock should be correct because Publication has non mockable clock? 

`syncUninjectableModifiedDates` just adds confusion and is handled duplicating what is being done in `expectedDoiRequestSummary`?

Reply:
fixed